cmd/contour/ingressstatus.go

```diff
@@ -81,9 +82,10 @@ func (isw *loadBalancerStatusWriter) Start(stop <-chan struct{}) error {
 			inf := factory.Networking().V1beta1().Ingresses().Informer()
 			log := isw.log.WithField("context", "IngressStatusUpdater")
 			inf.AddEventHandler(&k8s.IngressStatusUpdater{
-				Client: isw.clients.ClientSet(),
-				Logger: log,
-				Status: lbs,
+				Client:       isw.clients.ClientSet(),
```

Comment:
Add another log field above on line 83 to capture the ingress class we are matching against (can't use the `ingress-class` field name since that's already in use.

Reply:
Good point, thanks.